app/src/androidTest/java/com/rlab/sejima/MUHeaderTests.java

```diff
@@ -5,109 +5,96 @@
 import android.graphics.Typeface;
 import android.support.test.InstrumentationRegistry;
 import android.support.test.runner.AndroidJUnit4;
-
 import android.util.DisplayMetrics;
+import android.widget.RelativeLayout;
 
 import com.rlab.sejima.features.MUHeader;
 
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-
-import static android.view.View.TEXT_ALIGNMENT_TEXT_END;
-import static android.view.View.TEXT_ALIGNMENT_TEXT_START;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 
 @RunWith(AndroidJUnit4.class)
 public class MUHeaderTests {
 
     private Context mContext;
+    private MUHeader mMUHeader;
 
     @Before
     public void setUp() {
         mContext= InstrumentationRegistry.getTargetContext();
+        mMUHeader= new MUHeader(mContext);
+        assertNotNull(mMUHeader);
     }
 
     @Test
     public void defaultValues() {
 
-        MUHeader muHeader = new MUHeader(mContext);
-        assertNotNull(muHeader);
-
+        float scale = (float) mContext.getResources().getDisplayMetrics().densityDpi / DisplayMetrics.DENSITY_DEFAULT;
```

Comment:
Can be a private var and set in the setUp() function.

Reply:
sure.